tf_environment.yml

```diff
@@ -0,0 +1,8 @@
+name: pyvertical-dev-tf
+channels:
+  - tensorflow
+dependencies:
+  - python=3.8
+  - numpy >=1.16
```

Comment:
this environment should have `jupyterlab` so people can run the notebooks without doing anything other than installing this conda environment

Reply:
Great, should we do the same for the PyTorch's environment.yml, instead of running `conda install notebook`?

Reply:
I forgot that I left it out for a reason - syft wasn't playing well with jupyterlab/notebook when conda was trying to resolve versions. Leave `jupyterlab` out for now